lib/index.js

```diff
@@ -1413,5 +1426,12 @@ exports.diagnoses = internals.validate.diagnoses = (function () {
 
 exports.normalize = internals.normalize = function (email) {
 
+    // $lab:coverage:off$
+    if (process.version[1] === '4' && email.match(/\0/g)) {
```

Comment:
I would recommend replacing `regex` with a simple `indexOf` for performance reasons.

`if (process.version[1] === '4' && email.indexOf('/u0000') === 0) {`